indexer/bag_indexer/src/test/scala/uk/ac/wellcome/platform/archive/indexer/bag/models/IndexedStorageManifestTest.scala

```diff
@@ -0,0 +1,119 @@
+package uk.ac.wellcome.platform.archive.indexer.bag.models
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import uk.ac.wellcome.platform.archive.common.bagit.models.BagVersion
+import uk.ac.wellcome.platform.archive.common.generators.{
+  IngestGenerators,
+  StorageManifestGenerators
+}
+import uk.ac.wellcome.platform.archive.common.storage.services.DestinationBuilder
+import uk.ac.wellcome.platform.archive.indexer.bags.models.{
+  IndexedFileFields,
+  IndexedPayloadStats,
+  IndexedStorageManifest,
+  IndexedSuffixTally
+}
+
+class IndexedStorageManifestTest
+    extends AnyFunSpec
+    with Matchers
+    with IngestGenerators
+    with StorageManifestGenerators {
```

Comment:
How about a short test that it pulls through other attributes like "id" and "space"?